restart.sh

```diff
@@ -0,0 +1,16 @@
+
+#!/bin/bash
+set -euo pipefail
+
+export KUBE_NAMESPACE=${ENVIRONMENT}
+
+if [[ ${KUBE_NAMESPACE} == *prod ]]
+then
+    export KUBE_SERVER="https://kube-api-prod.prod.acp.homeoffice.gov.uk"
```

Comment:
```suggestion
    export KUBE_SERVER="https://kube-api-prod.prod.acp.homeoffice.gov.uk"
    export KUBE_CERTIFICATE_AUTHORITY="https://raw.githubusercontent.com/UKHomeOffice/acp-ca/master/acp-prod.crt"
```

Maybe we should gate this: `if (prod) quit()`.

Reply:
I have added the certificate authority - wasn't sure it was required.